2023-10-16 16:03:00.998::::::ERROR:::<nil>:::69:::recover.go.36 gin.CreateGin.PanicRecover.func1.1:::In PanicRecover,Error:ERR Error running script (call to f_57e87c619281ffd34e9bc44b56760057746c9d10): @user_script:2: user_script:2: attempt to compare number with nil 

2023-10-16 16:03:01::::::ERROR:::f69fd28a-614e-430c-9990-1f0ea2be210d:::69:::recover.go.49 gin.CreateGin.PanicRecover.func1.1:::panic stack info goroutine 69 [running]:
github.com/niuniumart/gosdk/gin.CreateGin.PanicRecover.func1.1()
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/recover/recover.go:47 +0x21b
panic({0x14a6f40?, 0xc0001ec440?})
	C:/Program Files/Go/src/runtime/panic.go:914 +0x21f
blitzSeckill/admin/src/cache.Evalsha({0x1598a47?, 0xc000621588?}, 0xcc0d45?)
	D:/project/Go/blitzSeckill/admin/src/cache/redis.go:62 +0x128
blitzSeckill/admin/src/controller/activity.SecKill(0xc000072c00?)
	D:/project/Go/blitzSeckill/admin/src/controller/activity/activity.go:69 +0x2d
github.com/gin-gonic/gin.(*Context).Next(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.Cors.func3(0xc000008e40?)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/cors/cors.go:16 +0x4f
github.com/gin-gonic/gin.(*Context).Next(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.InfoLog.func2(0xc000618100)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/logprint/logprint.go:96 +0x57c
github.com/gin-gonic/gin.(*Context).Next(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.PanicRecover.func1(0x113936e?)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/recover/recover.go:56 +0x45
github.com/gin-gonic/gin.(*Context).Next(...)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.CustomRecoveryWithWriter.func1(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/recovery.go:102 +0x7a
github.com/gin-gonic/gin.(*Context).Next(...)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.LoggerWithConfig.func1(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/logger.go:240 +0xde
g


2023-10-16 16:03:07.306::::::ERROR:::<nil>:::57:::recover.go.36 gin.CreateGin.PanicRecover.func1.1:::In PanicRecover,Error:ERR Error running script (call to f_57e87c619281ffd34e9bc44b56760057746c9d10): @user_script:2: user_script:2: attempt to compare number with nil 

2023-10-16 16:03:07.307::::::ERROR:::8eed7b1d-0864-4015-a668-11fcc9c37202:::57:::recover.go.49 gin.CreateGin.PanicRecover.func1.1:::panic stack info goroutine 57 [running]:
github.com/niuniumart/gosdk/gin.CreateGin.PanicRecover.func1.1()
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/recover/recover.go:47 +0x21b
panic({0x14a6f40?, 0xc0001ec0a0?})
	C:/Program Files/Go/src/runtime/panic.go:914 +0x21f
blitzSeckill/admin/src/cache.Evalsha({0x1598a47?, 0xc000621588?}, 0xcc0d45?)
	D:/project/Go/blitzSeckill/admin/src/cache/redis.go:62 +0x128
blitzSeckill/admin/src/controller/activity.SecKill(0xc000580000?)
	D:/project/Go/blitzSeckill/admin/src/controller/activity/activity.go:69 +0x2d
github.com/gin-gonic/gin.(*Context).Next(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.Cors.func3(0xc00060c018?)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/cors/cors.go:16 +0x4f
github.com/gin-gonic/gin.(*Context).Next(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.InfoLog.func2(0xc000618100)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/logprint/logprint.go:96 +0x57c
github.com/gin-gonic/gin.(*Context).Next(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.PanicRecover.func1(0x4db100?)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/recover/recover.go:56 +0x45
github.com/gin-gonic/gin.(*Context).Next(...)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.CustomRecoveryWithWriter.func1(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/recovery.go:102 +0x7a
github.com/gin-gonic/gin.(*Context).Next(...)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.LoggerWithConfig.func1(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/logger.go:240 +0xde
gi


2023-10-16 16:03:10.163::::::ERROR:::<nil>:::46:::recover.go.36 gin.CreateGin.PanicRecover.func1.1:::In PanicRecover,Error:ERR Error running script (call to f_57e87c619281ffd34e9bc44b56760057746c9d10): @user_script:2: user_script:2: attempt to compare number with nil 

2023-10-16 16:03:10.164::::::ERROR:::47e8ebc6-b2c3-4bdf-84f1-67c63e345afa:::46:::recover.go.49 gin.CreateGin.PanicRecover.func1.1:::panic stack info goroutine 46 [running]:
github.com/niuniumart/gosdk/gin.CreateGin.PanicRecover.func1.1()
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/recover/recover.go:47 +0x21b
panic({0x14a6f40?, 0xc0003162a0?})
	C:/Program Files/Go/src/runtime/panic.go:914 +0x21f
blitzSeckill/admin/src/cache.Evalsha({0x1598a47?, 0xc00061d588?}, 0xcc0d45?)
	D:/project/Go/blitzSeckill/admin/src/cache/redis.go:62 +0x128
blitzSeckill/admin/src/controller/activity.SecKill(0xc000072c00?)
	D:/project/Go/blitzSeckill/admin/src/controller/activity/activity.go:69 +0x2d
github.com/gin-gonic/gin.(*Context).Next(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.Cors.func3(0xc000008600?)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/cors/cors.go:16 +0x4f
github.com/gin-gonic/gin.(*Context).Next(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.InfoLog.func2(0xc000618100)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/logprint/logprint.go:96 +0x57c
github.com/gin-gonic/gin.(*Context).Next(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.PanicRecover.func1(0x113936e?)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/recover/recover.go:56 +0x45
github.com/gin-gonic/gin.(*Context).Next(...)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.CustomRecoveryWithWriter.func1(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/recovery.go:102 +0x7a
github.com/gin-gonic/gin.(*Context).Next(...)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.LoggerWithConfig.func1(0xc000618100)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/logger.go:240 +0xde
g


2023-10-16 16:03:42.206::::::ERROR:::<nil>:::26:::recover.go.36 gin.CreateGin.PanicRecover.func1.1:::In PanicRecover,Error:ERR Error running script (call to f_57e87c619281ffd34e9bc44b56760057746c9d10): @user_script:2: user_script:2: attempt to compare number with nil 

2023-10-16 16:03:42.207::::::ERROR:::55b3861c-5f4b-4ec6-8547-3b1cc51ffb3b:::26:::recover.go.49 gin.CreateGin.PanicRecover.func1.1:::panic stack info goroutine 26 [running]:
github.com/niuniumart/gosdk/gin.CreateGin.PanicRecover.func1.1()
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/recover/recover.go:47 +0x21b
panic({0x886f40?, 0xc0002821b0?})
	C:/Program Files/Go/src/runtime/panic.go:914 +0x21f
blitzSeckill/admin/src/cache.Evalsha({0x978a47?, 0xc000279588?}, 0xa0d45?)
	D:/project/Go/blitzSeckill/admin/src/cache/redis.go:62 +0x128
blitzSeckill/admin/src/controller/activity.SecKill(0x1019d80?)
	D:/project/Go/blitzSeckill/admin/src/controller/activity/activity.go:69 +0x2d
github.com/gin-gonic/gin.(*Context).Next(0xc0004ca200)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.Cors.func3(0xc000008348?)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/cors/cors.go:16 +0x4f
github.com/gin-gonic/gin.(*Context).Next(0xc0004ca200)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.InfoLog.func2(0xc0004ca200)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/logprint/logprint.go:96 +0x57c
github.com/gin-gonic/gin.(*Context).Next(0xc0004ca200)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.PanicRecover.func1(0x51936e?)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/recover/recover.go:56 +0x45
github.com/gin-gonic/gin.(*Context).Next(...)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.CustomRecoveryWithWriter.func1(0xc0004ca200)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/recovery.go:102 +0x7a
github.com/gin-gonic/gin.(*Context).Next(...)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.LoggerWithConfig.func1(0xc0004ca200)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/logger.go:240 +0xde
github.c


2023-10-16 16:07:11.341::::::ERROR:::<nil>:::38:::recover.go.36 gin.CreateGin.PanicRecover.func1.1:::In PanicRecover,Error:ERR Error running script (call to f_57e87c619281ffd34e9bc44b56760057746c9d10): @user_script:2: user_script:2: attempt to compare number with nil 

2023-10-16 16:07:11.341::::::ERROR:::6f7522dc-8d53-48fc-9d80-776d4bc7bb92:::38:::recover.go.49 gin.CreateGin.PanicRecover.func1.1:::panic stack info goroutine 38 [running]:
github.com/niuniumart/gosdk/gin.CreateGin.PanicRecover.func1.1()
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/recover/recover.go:47 +0x21b
panic({0x11a6f40?, 0xc0001e4800?})
	C:/Program Files/Go/src/runtime/panic.go:914 +0x21f
blitzSeckill/admin/src/cache.Evalsha({0x1298a47?, 0xc0000d1588?}, 0x9c0d45?)
	D:/project/Go/blitzSeckill/admin/src/cache/redis.go:62 +0x128
blitzSeckill/admin/src/controller/activity.SecKill(0xc000072c00?)
	D:/project/Go/blitzSeckill/admin/src/controller/activity/activity.go:69 +0x2d
github.com/gin-gonic/gin.(*Context).Next(0xc000120400)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.Cors.func3(0xc00028c018?)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/cors/cors.go:16 +0x4f
github.com/gin-gonic/gin.(*Context).Next(0xc000120400)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.InfoLog.func2(0xc000120400)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/logprint/logprint.go:96 +0x57c
github.com/gin-gonic/gin.(*Context).Next(0xc000120400)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.PanicRecover.func1(0xe3936e?)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/recover/recover.go:56 +0x45
github.com/gin-gonic/gin.(*Context).Next(...)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.CustomRecoveryWithWriter.func1(0xc000120400)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/recovery.go:102 +0x7a
github.com/gin-gonic/gin.(*Context).Next(...)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.LoggerWithConfig.func1(0xc000120400)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/logger.go:240 +0xde
gi


2023-10-16 16:25:37.624::::::ERROR:::<nil>:::29:::recover.go.36 gin.CreateGin.PanicRecover.func1.1:::In PanicRecover,Error:ERR Error running script (call to f_57e87c619281ffd34e9bc44b56760057746c9d10): @user_script:2: user_script:2: attempt to compare number with nil 

2023-10-16 16:25:37.626::::::ERROR:::cb5528a9-b743-401c-a826-656e4d69480e:::29:::recover.go.49 gin.CreateGin.PanicRecover.func1.1:::panic stack info goroutine 29 [running]:
github.com/niuniumart/gosdk/gin.CreateGin.PanicRecover.func1.1()
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/recover/recover.go:47 +0x21b
panic({0x1096f40?, 0xc0002229d0?})
	C:/Program Files/Go/src/runtime/panic.go:914 +0x21f
blitzSeckill/admin/src/cache.Evalsha({0x1188a47?, 0xc000165588?}, 0x8b0d45?)
	D:/project/Go/blitzSeckill/admin/src/cache/redis.go:62 +0x128
blitzSeckill/admin/src/controller/activity.SecKill(0xc000580000?)
	D:/project/Go/blitzSeckill/admin/src/controller/activity/activity.go:69 +0x2d
github.com/gin-gonic/gin.(*Context).Next(0xc000526a00)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.Cors.func3(0xc0001de270?)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/cors/cors.go:16 +0x4f
github.com/gin-gonic/gin.(*Context).Next(0xc000526a00)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.InfoLog.func2(0xc000526a00)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/logprint/logprint.go:96 +0x57c
github.com/gin-gonic/gin.(*Context).Next(0xc000526a00)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.PanicRecover.func1(0xc00020e2a0?)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/recover/recover.go:56 +0x45
github.com/gin-gonic/gin.(*Context).Next(...)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.CustomRecoveryWithWriter.func1(0xc000526a00)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/recovery.go:102 +0x7a
github.com/gin-gonic/gin.(*Context).Next(...)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.LoggerWithConfig.func1(0xc000526a00)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/logger.go:240 +0xd


2023-10-16 16:30:21.911::::::ERROR:::<nil>:::39:::recover.go.36 gin.CreateGin.PanicRecover.func1.1:::In PanicRecover,Error:ERR Error running script (call to f_57e87c619281ffd34e9bc44b56760057746c9d10): @user_script:2: user_script:2: attempt to compare number with nil 

2023-10-16 16:30:21.912::::::ERROR:::4bede141-4179-4789-bbc3-68745c95113e:::39:::recover.go.49 gin.CreateGin.PanicRecover.func1.1:::panic stack info goroutine 39 [running]:
github.com/niuniumart/gosdk/gin.CreateGin.PanicRecover.func1.1()
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/recover/recover.go:47 +0x21b
panic({0xb86f40?, 0xc00061a070?})
	C:/Program Files/Go/src/runtime/panic.go:914 +0x21f
blitzSeckill/admin/src/cache.Evalsha({0xc78a47?, 0xc00043d588?}, 0x3a0d45?)
	D:/project/Go/blitzSeckill/admin/src/cache/redis.go:64 +0x18b
blitzSeckill/admin/src/controller/activity.SecKill(0xc000480000?)
	D:/project/Go/blitzSeckill/admin/src/controller/activity/activity.go:69 +0x2d
github.com/gin-gonic/gin.(*Context).Next(0xc0000a0200)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.Cors.func3(0xc000008510?)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/cors/cors.go:16 +0x4f
github.com/gin-gonic/gin.(*Context).Next(0xc0000a0200)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.InfoLog.func2(0xc0000a0200)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/logprint/logprint.go:96 +0x57c
github.com/gin-gonic/gin.(*Context).Next(0xc0000a0200)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 +0x2b
github.com/niuniumart/gosdk/gin.CreateGin.PanicRecover.func1(0x4ac9b0?)
	D:/go_path/pkg/mod/github.com/niuniumart/gosdk@v0.0.0-20230802040621-c42b9c9fc9bd/middleware/recover/recover.go:56 +0x45
github.com/gin-gonic/gin.(*Context).Next(...)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.CustomRecoveryWithWriter.func1(0xc0000a0200)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/recovery.go:102 +0x7a
github.com/gin-gonic/gin.(*Context).Next(...)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.LoggerWithConfig.func1(0xc0000a0200)
	D:/go_path/pkg/mod/github.com/gin-gonic/gin@v1.9.1/logger.go:240 +0xde
gith


